pytest_rpc.py

```diff
@@ -54,3 +54,25 @@ def pytest_collection_modifyitems(items):
         if marker is not None:
             test_id = marker.args[0]
             item.user_properties.append(('test_id', test_id))
+
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_runtest_setup(item):
+    """Add XML properties group to the 'testcase' element that captures start time in UTC.
+
+    Args:
+        items (list(_pytest.nodes.Item)): List of item objects.
+    """
+    now = datetime.utcnow()
```

Comment:
The qTest API doesn't like milliseconds on the timestamp so you're going to want to use timestamp formatting:

```python
now = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
```